code/models/projection_helper_functions.R

```diff
@@ -231,9 +258,42 @@ compute_predicted_cum_cases <- function(full_data, model, policy_variables_used,
           unit_population = population[1],
           proportion_confirmed = proportion_confirmed,
           all = TRUE
-        ))
-      }) %>% 
-      unnest(projection_output)
+        ) %>%
+          rename_all(~paste0(., "_no_policy")))
+      }) %>%
+      ungroup() %>%
+      left_join(datetimes, by = datetimes %>% names() %>% str_subset("^adm[0-2]_name$")) %>%
+      unnest(c(projection_output, timestep, date)) %>%
```

Comment:
@kendonB can i take it on faith that this line does what you want it to do?

Reply:
yip I'm happy with this one - can talk it through if you want